docs/detections/osquery-response-action.asciidoc

```diff
@@ -0,0 +1,62 @@
+[[osquery-response-action]]
+=== Add Osquery Response Actions
+Osquery Response Actions allow you to add live queries to custom query rules so you can automatically collect data on systems the rule is monitoring. Use this data to support your alert triage and investigation efforts.
+
+NOTE: Response Actions are in technical preview.
+
+[role="screenshot"]
+image::images/available-response-actions.png[Shows available response actions]
+
+.Requirements
+[sidebar]
+--
+* Osquery Response Actions are available with a https://www.elastic.co/pricing[Platinum or Enterprise subscription].
+* The {kibana-ref}/manage-osquery-integration.html[Osquery manager integration] must be installed.
+* {agent}'s {fleet-guide}/view-elastic-agent-status.html[status] must be `Healthy`. Refer to {fleet-guide}/fleet-troubleshooting.html[{fleet} Troubleshooting] if it isn't.
+* Your role must have {kibana-ref}/osquery.html[Osquery feature privileges].
+--
+
+[float]
+[[add-osquery-response-action]]
+=== Add Osquery Response Actions to rules
+
+You can add Osquery Response Actions to new or existing custom query rules. Queries are ran every time the rule executes.
+
+. Choose one of the following:
+** *New rule*: When you are on the last step of <<create-custom-rule,custom query rule>>, go to the Response Actions section and click the *osquery* icon.
+** *Existing rule*: Edit the rule's settings, then go to the *Actions* tab. In the tab, click the *osquery* icon under the Response Actions section.
+. Specify whether you want to set up a single live query or a pack:
+** *Query*: Select a saved query or enter a new one in the text box. After you enter the query, you can expand the **Advanced** section to view or set {kibana-ref}/osquery.html#osquery-map-fields[mapped ECS fields] included in the results from the live query. Mapping ECS fields is optional.
+** *Pack*: Select from query packs that have been loaded and activated. After you select a pack, all of the queries in the pack are displayed.
++
+TIP: Refer to {kibana-ref}/osquery.html#osquery-prebuilt-packs-queries[prebuilt packs] to learn about using and managing Elastic prebuilt packs.
++
+[role="screenshot"]
+image::images/setup-single-query.png[Shows how to set up a single query]
++
+
+. Click the *osquery* icon to add more live queries (optional).
+. Create the rule or save your changes to finish adding the queries.
```

Comment:
Unclear what's the difference between "create the rule" and "save your changes" -- at this point in creating/editing a rule, the only thing the user can do to proceed is click "Save changes".

Reply:
@joepeeples at the core, there's not a difference between creating a new rule and saving changes to an existing one that you're updating. That said, I did want to list both options because the action is slightly different in the UI. For example when creating a new rule, users would click **Create & enable rule** (or **Create rule without enabling it**) to save the rule. When editing, an existing rule, they'd save their changes by clicking **Save changes**.

Reply:
@nastasha-solomon Gotcha, thanks for explaining. The rest of the procedure is nicely specific about what the user should do, click, etc., and this step seemed less clear on what exactly to do. Would this work?
```suggestion
. Click **Create & enable rule** (for a new rule) or **Save changes** (for existing rules) to finish adding the queries.
```